pandas/core/generic.py

```diff
@@ -8871,8 +8871,15 @@ def _add_series_only_operations(cls):
         axis_descr, name, name2 = _doc_parms(cls)
 
         def nanptp(values, axis=0, skipna=True):
+            """
+            .. deprecated:: 0.24.0
+                Use numpy.ptp instead
```

Comment:
I am not sure this is the right format for the doc-str, cc @jorisvandenbossche

Reply:
Thanks - will wait for confirmation on right format from @jorisvandenbossche 

Based on the other deprecation PRs I checked, this seemed to be the expected format - for other messages I had noted earlier I got Travis failures (example below from different PR submitted)
```python
Check for deprecated messages without sphinx directive
pandas/core/indexes/multi.py:        DEPRECATED. to_hierarchical will be removed in a future version.
Check for deprecated messages without sphinx directive DONE
...
The command "ci/lint.sh" exited with 1.
```

Reply:
Yes, this is the correct format. But, shouldn't this rather be in `cls.ptp` docstring passed to `_make_stat_function` below? 
(didn't really check, but make sure that `pd.Series.ptp?` gives the correct docstring)

Reply:
Did a quick check, and indeed adding the deprecation message does not update the actual docstring of `Series.ptp`, you need to add it to the argument passed to `_make_stat_function` a few lines below